src/Makefile.test.include

```diff
@@ -256,6 +255,7 @@ test_fuzz_fuzz_SOURCES = \
  test/fuzz/checkqueue.cpp \
  test/fuzz/coins_view.cpp \
  test/fuzz/coinscache_sim.cpp \
+ test/fuzz/compression_roundtrip.cpp \
```

Comment:
In 1d376496fe9ac7110d0fac773ae5b5d9ef32aa59:

I think it would make sense to split this into two commits; one that adds the code and one that adds fuzztests.

The easiest way to do this is to

1. use `git rebase -i`, edit this commit, delete all the fuzzing stuff (remembering to update Makefile.test.include)
2. commit that.
3. revert it with `git revert HEAD`, then `git rebase --continue`.
4. Then `rebase -i` again, squashing the "delete fuzztests" commit into this, and rewording the revert one to "add fuzztests".

Reply:
Did this for the rpc call too:
git rebase master -i
git reset HEAD^
Added primitive stuff
git commit -m
Added rpc stuff
git commit -m
Added fuzz stuff
git commit -m
git rebase --continue